package.json

```diff
@@ -78,7 +78,8 @@
     "scroll": "^3.0.1",
     "serve-handler": "^6.1.2",
     "slick-carousel": "^1.8.1",
-    "unist-util-visit": "2.0.2"
+    "unist-util-visit": "2.0.2",
+    "upath": "^1.2.0"
```

Comment:
not sure ... but should it be part of the devDeps section?

Reply:
I suppose it would fit more in the current setup, but technically this is the right place for it and most of those devDependencies should be moved up to regular dependencies. It makes no difference in the practical sense as the switch to `YARN_PRODUCTION=false` means we keep all deps and devDeps in the same way on deploy.

Reply:
yep, you are right. For the website there are no difference at all, right? 

may be we can keep linters, tests, and other stuff there? may be should move them to optionalDeps? to avoid installing them on Heroku?

Reply:
anyway, merging this ... since this one can stay in deps.